docs/source/tracking.rst

```diff
@@ -429,6 +429,41 @@ Autologging captures the following information:
++------------------------------------------+------------------------------------------------------------+-------------------------------------------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------+
+| ``trainer.callbacks.earlystopping``      | Training loss; validation loss;average_test_accuracy;      | ``fit()`` parameters; optimizer name; learning rate; epsilon.                       | --            | Model summary on training start; `MLflow Model <https://mlflow.org/docs/latest/models.html>`_ (Pytorch model) on training end;                |
+|                                          | user-defined-metrics.                                      |                                                                                     |               | Also facilitates logging of torchscript models;                                                                                               |
+|                                          | Metrics from the ``EarlyStopping`` callbacks.              | Parameters associated with ``EarlyStopping``.                                       |               | Optional Documents such as ExtraFiles,requirements.txt on Training End could also be logged;                                                  |
+|                                          | For example, ``stopped_epoch``, ``restored_epoch``,        | For example, ``min_delta``, ``patience``, ``baseline``,                             |               | Best Pytorch model  Checkpoint if training stops due to early stopping callback.                                                              |
+|                                          | ``restore_best_weight``, etc.                              | ``restore_best_weights``, etc                                                       |               |                                                                                                                                               |
++------------------------------------------+------------------------------------------------------------+-------------------------------------------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------+
+| ``pytorch-lightning trainer``            |Training loss;validation loss;average_test_accuracy;        | ``fit()`` parameters; optimizer name; learning rate; epsilon                        | --            | Model summary on training start; `MLflow Model <https://mlflow.org/docs/latest/models.html>`_ (Pytorch model) on training end;                |
```

Comment:
@karthik-77 Please change the sequence of the table rows. Move the `pytorch-lightning trainer` row to be the 1st row in the table

Reply:
I have changed the order. First row is the pytorch lightning trainer and the callback comes next,